CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Display timezone for session timeout in user menu [#1764](https://github.com/greenbone/gsa/pull/1764)
 
 ### Changed
+- Changed report TlsCertificate table headers to match TlsCertificate assets
+table headers [#2042](https://github.com/greenbone/gsa/pull/2042)
```

Comment:
```suggestion
table headers [#2044](https://github.com/greenbone/gsa/pull/2044)
```

Reply:
Ok, now this seems like I had a small stroke...